### PR TITLE
refactor: farewell to flash wreckage

### PR DESCRIPTION
**Pull request by syncsyncsynchalt**
Now he has no job other than to show WARNING on startup.
It's time to say goodbye.

### CODE REVIEW

**Comment by KagamiChan** [comment]
Is there any DMM game that is still using Flash?

as we do support other games on DMM, we need to keep it for those Flash games

**Comment by syncsyncsynchalt** [comment]
I've looked at the DMM games listed in ./lib/proxy.es
I feel like I can turn it off.

- Kanpani: HTML5(2019/12-)
- ShiroPro: HTML5(2016/3-)
- Shinken: closed(-2017/9)
- FlowerKnightGirls: HTML5(2019/6-)
- ToukenRanbu: HTML5(2019/7-)
- KanColle: HTML5(2018/8-)

**Comment by KochiyaOcean** [comment]
I think it's okay to deprecate flash5 support since all DMM games using flash has refactored to html5 or end their service

File: views/services/utils.es [hunk 1]
```diff
@@ -67,21 +67,13 @@ export const gameReload = () => {
   } else {
     doc = document;
   }
-  var flash = doc.getElementById('flashWrap');
```
Comment (KochiyaOcean) [inline views/services/utils.es:70]: These codes are still required for game reload function working, but it's okay to rename these variable names.

Comment (syncsyncsynchalt) [reply views/services/utils.es:70]: Thx for pointing that out. I broke the game reload feature.
I fixed it and checked the operation.

**Pull request merged**